imagegw/shifter_imagegw/dockerv2.py

```diff
@@ -383,7 +383,10 @@ def do_token_auth(self, auth_loc_str, creds=False):
                              'failed to get auth connection')
 
         headers = {}
-        if creds and self.username is not None and self.password is not None:
+        if self.username=='$oauthtoken':
+            self.private = True
+            headers['Authoriation'] = 'Bearer %s' % (self.password)
```

Comment:
authoriation is probably a typo

Reply:
the password is just sent plaintext?  not even the base64 encoded to assure special characters don't derail the whole thing?

Reply:
yeah it is just a bearer token of some sort.  Good catch on the misspelling.  I'm confused as to how this worked now.  Let me test this again.

Reply:
It winds up the typo was part of why it worked.  I need to figure out what is going on here.